ui/src/app/zones/views/ZonesListForm/ZonesListForm.tsx

```diff
@@ -51,13 +52,13 @@ const ZonesListForm = ({ closeForm }: Props): JSX.Element => {
       <Row>
         <Col size="6">
           <FormikField
-            component={Input}
             label="Name"
             placeholder="Name"
             type="text"
             name="name"
             required
           />
+          <FormikField label="Description" type="text" name="description" />
```

Comment:
I've added a Description form field here - seemed strange that this wasn't always the case. But like we've said before, if the angular app looks like garbage or it feels like something could be improved, don't feel like you have to implement things 1:1